elasticsearch/src/main/java/com/yahoo/ycsb/db/ElasticSearchClient.java

```diff
@@ -90,13 +92,15 @@ public void init() throws DBException {
         .parseBoolean(props.getProperty("elasticsearch.remote", "false"));
     Boolean newdb =
         Boolean.parseBoolean(props.getProperty("elasticsearch.newdb", "false"));
-    Builder settings = settingsBuilder().put("node.local", "true")
+    Builder settings = Settings.settingsBuilder()
+        .put("node.local", "true")
```

Comment:
Maybe I'm missing context due to the diff view. Let me try loading in a non-mobile client....